app/bundles/CoreBundle/Helper/InputHelper.php

```diff
@@ -345,8 +345,10 @@ public static function email($value, $urldecode = false)
         }
 
         $value = substr($value, 0, 254);
+        $value = filter_var($value, FILTER_SANITIZE_EMAIL);
+        $value = str_replace('..', '.', $value);
```

Comment:
Not sure what was the reason, but this test fails:
```
$clean = InputHelper::email('john...doe@email.com');
$this->assertEquals('john.doe@email.com', $clean);
```

Reply:
Not sure why. It passes for me and for Travis https://travis-ci.org/mautic/mautic/builds/285801327

Reply:
Yes, tests in this PR passes because there is not test like this, you have to add it to fail :)
But I think this is not so important now